config.toml

```diff
@@ -21,6 +21,7 @@ ignoreFiles = [
 
 [params]
     google_analytics_id="UA-133584243-1"
+    version="0.19.x"
```

Comment:
this is the current version used for the canonical URLs, as soon as 1.0.x is live we need to adapt this string

Reply:
Cool.  Let me note this down -- when i do the PR that releases the 1.0 docs, I'll include this.